econml/validate/drtester.py

```diff
@@ -409,15 +409,19 @@ def evaluate_cal(
             # Calculate R-square calibration score
             cal_r_squared[k] = 1 - (cal_score_g / cal_score_o)
 
-            df_plot1 = pd.DataFrame({'ind': np.array(range(n_groups)),
-                                     'gate': gate, 'se_gate': se_gate,
-                                    'g_cate': g_cate, 'se_g_cate': se_g_cate})
-            df_plot1['tmt'] = self.treatments[k + 1]
-            df_plot = pd.concat((df_plot, df_plot1))
+            df_plot1 = pd.DataFrame({
+                'ind': np.array(range(n_groups)),
+                'gate': gate,
+                'se_gate': se_gate,
+                'g_cate': g_cate,
+                'se_g_cate': se_g_cate
+            })
+
+            plot_dict[self.treatments[k + 1]] = df_plot1
```

Comment:
I think now that there's not a naming conflict, just `df_plot` instead of `df_plot1` would be a better variable name:

```suggestion
            df_plot = pd.DataFrame({
                'ind': np.array(range(n_groups)),
                'gate': gate,
                'se_gate': se_gate,
                'g_cate': g_cate,
                'se_g_cate': se_g_cate
            })

            plot_dict[self.treatments[k + 1]] = df_plot
```

Reply:
Done, addressed in latest commits.